layouts/shortcodes/fields/latinx.html

```diff
@@ -0,0 +1,14 @@
+<div class="field">
+  <label class="label">Do you identify as a Latinx / a person of Latin American descent?*</label>
```

Comment:
i would remove a Latinx so it's just Latinx

do you identify as Latina

Reply:
updated